scout/utils/ensembl_rest_clients.py

```diff
@@ -43,19 +45,22 @@ def send_request(url):
         Returns:
             data(dict): dictionary from json response
         """
-        data = {}
+        error = None
+        data = None
         try:
             response = requests.get(url, headers=HEADERS)
-            if response.status_code == 404:
-                LOG.info("Request failed for url %s\n", url)
-                response.raise_for_status()
-            data = response.json()
+            if response.status_code not in [404, 500]:
+
+                data = response.json()
+            else:
+                error = f"Ensembl request failed with code:{response.status_code} for url {url}"
         except requests.exceptions.MissingSchema as err:
-            LOG.info("Request failed for url %s: Error: %s\n", url, err)
-            data = err
+            error = f"Ensembl request failed with MissingSchema error for url {url}"
```

Comment:
Does the exception hold anything more than the type here? If so consider adding to the formatted error string, else no need to capture it as unused variable `err`.

Reply:
Actually no, so I'll just go and remove that err, good point!